src/main/java/com/thealgorithms/sorts/SelectionSort.java

```diff
@@ -1,46 +1,38 @@
 package com.thealgorithms.sorts;
 
+/**
+ * This class provides an implementation of the selection sort algorithm.
+ * It sorts an array of elements in increasing order using an iterative approach.
+ */
 public class SelectionSort implements SortAlgorithm {
-
     /**
-     * Generic selection sort algorithm in increasing order.
+     * Sorts an array of comparable elements in increasing order using the selection sort algorithm.
      *
-     * @param arr the array to be sorted.
-     * @param <T> the class of array.
-     * @return sorted array.
+     * @param array the array to be sorted
+     * @param <T> the class of array elements
+     * @return the sorted array
      */
     @Override
-    public <T extends Comparable<T>> T[] sort(T[] arr) {
-        int n = arr.length;
-        for (int i = 0; i < n - 1; i++) {
+    public <T extends Comparable<T>> T[] sort(T[] array) {
+        if (array.length == 0) {
+            return array;
+        }
+
```

Comment:
```suggestion
```

Reply:
This is already removed according to this comment https://github.com/TheAlgorithms/Java/pull/5265#discussion_r1659318388